static/js/global_init.js

```diff
@@ -24,6 +25,18 @@ if (!Object.entries) {
   entries.shim();
 }
 
+let sandbox;
+before(() => { // eslint-disable-line mocha/no-top-level-hooks
+  sandbox = sinon.sandbox.create();
+  sandbox.stub(HTMLCanvasElement.prototype, 'getContext').returns({
```

Comment:
Just curious why this needs to be here in `global_init.js` and not `VideoPlayer_test.js`.  I moved this code block there to see if it would work and tests failed, but don't quite understand why.

Reply:
I wanted to have the mock in place for both `VideoPlayer` and `VideoSubtitleCard`. Regarding `before` instead of `beforeEach`, I wanted it to handle the `setTimeout` callback which might execute between tests. It is kind of hacky but I didn't want to edit the functionality in this PR, just the tests